applications/nrf_desktop/src/events/Kconfig

```diff
@@ -62,6 +62,10 @@ config DESKTOP_INIT_LOG_HID_REPORT_SENT_EVENT
 	bool "HID report sent event"
 	default y
 
+config DESKTOP_INIT_LOG_KEYBOARD_LEDS_EVENT
+	bool "Keyboard LEDs event"
+	default y
```

Comment:
Why is it so popular not to document Kconfig settings ?

Those settings are rendered into docs, so please make a good description of the setting.

Settings are rendered here:
http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/nrf_desktop/README.html#cmdoption-arg-CONFIG_DESKTOP_INIT_LOG_MOTION_EVENT

(The link is to the settings below, as this PR has not built its docs yet)

Reply:
I would say this is a common problem for all the events in nrf_desktop.
We display a comment in menuconfig, so everything is clear there. Still, help would be useful in docs. I will add this as separate PR, ok ?

Reply:
> @tejlmand The justification for lack of help is that it often repeats the same words as in config prompt. If there is nothing to be added into help why adding it?

@pdunaj The prompt states: `Keyboard LEDs event`

Does that mean:
- A LED will toggle every time an event is received ?
- A Keyboard LED toggled will generate an event ?
- How do I control which LED(s) are used ?
  - Here the help text could be useful to direct the reader else where:
    For example: To configure the LEDs in use, please refer to the nRF Desktop documentation on how to bind the LEDs for your platform to an event. (Or whatever the user needs to know)

I see lot of possibilities to provide more detailed information to our users than just a dump `Enable feature X`.
- What does feature X
- What else should I know about feature X
- Are there dependencies to feature X that I should know about for better experince
- ....

Reply:
@MarekPieta where did all the Kconfig files go ?

Reply:
I applied changes requested during code review. Changes made to the Kconfig files were no longer needed and I removed them.

Reply:
> Changes made to the Kconfig files were no longer needed and I removed them.

Guess that also a way to get PR approval from a grumpy old Kconfig reviewer like me :laughing: